inc/html.class.php

```diff
@@ -1272,6 +1272,7 @@ static function includeHeader($title = '', $sector = 'none', $item = 'none', $op
 
       // End of Head
       echo "</head>\n";
+      self::glpi_flush();
```

Comment:
I'm not sure this one is really relevant.

Reply:
Once the browser has received the HTML head, it can begin to download all linked CSS files, favicons and other resources. These downloads can occur while the browser is idle and waiting for the main body content.

Reply:
OK, good point :)